### ISSUE TITLE
Attaching a large number of images to the report

### ISSUE THREAD

**Issue by crazyvan25** [jenkinsci/cucumber-reports-plugin#378]
Hi, is there a recommended approach to attach images to the report, in the case where the report has hundreds of scenarios and each one attaches an image (let's assume the average image size is 1-1.5MB)?

`scenario.embed(bytes, "image/png")`
produces a very large json file (i've generated a report file that is ~350MB) and the plugin throws an OutOfMemoryError when parsing it (i've tested this with a Jenkins heap size of 4GB). Thanks!

**Comment by damianszczepanik** [comment]
Well, I guess you're the first who has started to attach such big image and thus it might be a problem with Jenkins it self

**Comment by crazyvan25** [comment]
You are correct, the problem was with Jenkins. I assumed that the plugin would run on a Jennkins slave but it's running on the master (which is underprovisioned and only had about 150MB of memory available). 
Is the fact that it runs on the master intentional or should it be running on a slave?
